requirements_all.txt

```diff
@@ -1006,9 +1006,11 @@ python_opendata_transport==0.0.3
 python_openzwave==0.4.3
 
 # homeassistant.components.egardia
-# homeassistant.components.alarm_control_panel.egardia
 pythonegardia==1.0.38
```

Comment:
This line should be removed. The script should handle that automatically. Did you run the script?

Reply:
I ran the script.

Reply:
Oh wait, I think I know the problem. Will fix tomorrow.